love/src/components/AuthList/SummaryAuthList/SummaryAuthList.jsx

```diff
@@ -0,0 +1,461 @@
+                    ? 'Please insert items separated by commas with the required format'
+                    : ''}
+                </span>
+              </div>
+            </div>
+            <div className={styles.inputNewRequest}>
+              <Input
+                value={this.state.unauthorize_cscs}
+                placeholder="Insert items separated by commas"
+                onChange={(e) => this.setState({ unauthorize_cscs: this.formatInputText(e.target.value) })}
+              ></Input>
+              <div>
+                <span className={styles.wrongInput}>
+                  {this.state.wrong_input_cscs
+                    ? 'Please insert items separated by commas with the required format'
+                    : ''}
+                </span>
+              </div>
+            </div>
+            <div className={styles.textAreaNewRequest}>
+              <TextArea
+                value={this.state.message}
+                callback={this.handleMessageChange}
+                onKeyDown={() => null}
+                onKeyUp={() => null}
+              />
+            </div>
+            <div className={styles.inputNewRequest}>
+              <Input value={this.state.duration} onChange={this.handleDurationChange} />
+            </div>
+          </div>
+        </div>
+        <div className={styles.btnSend}>
+          <Button onClick={() => this.sendRequest()}>Commit New Authorization</Button>
+        </div>
+      </div>
+    );
+  };
+
+  sendRequest() {
+    const { user } = this.props;
+    const { csc_to_change, authorize_users, unauthorize_cscs, message, duration } = this.state;
+
+    this.setState({ wrong_input_users: false, wrong_input_cscs: false });
+
+    const passA = authorize_users.split(',').every((x) => x.charAt(0) === '+' || x.charAt(0) === '-');
+    const passB = unauthorize_cscs.split(',').every((x) => x.charAt(0) === '+' || x.charAt(0) === '-');
+
+    if (!passA) {
+      // send error message for authorized users
+      this.setState({ wrong_input_users: true });
+    }
+    if (!passB) {
+      // send error message for unauthorized cscs
+      this.setState({ wrong_input_cscs: true });
+    }
+    if (passA && passB) {
+      ManagerInterface.requestAuthListAuthorization(
+        user,
+        csc_to_change,
+        authorize_users,
+        unauthorize_cscs,
+        message,
+        duration,
+      ).then(() => {
+        this.setState({
+          csc_to_change: '',
+          authorize_users: '',
+          unauthorize_cscs: '',
+          message: '',
+          duration: '',
+          isActive: false,
+        });
+      });
+    }
+  }
+
+  render() {
+    const { authlistState, authlistAdminPermission } = this.props;
+    const {
+      selectedCSC,
+      selectedUser,
+      keywords,
+      cscOptions,
+      userOptions,
+      removeIdentityModalShown,
+      removeIdentityModalText,
+    } = this.state;
+
+    // console.log(authlistAdminPermission);
```

Comment:
This console.log must be removed

Reply:
I've removed this comment on further commits. Thanks!